src/Native/System.Net.Http.Native/pal_easy.h

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#include "pal_types.h"
+
+#include <curl/curl.h>
+
+#define CurlOptionLongBase 0
+#define CurlOptionObjectPointBase 10000
+#define CurlOptionFunctionPointBase 20000
+
```

Comment:
Again, my bias against macros here -- I'd use an unnamed enum to hold these values. You could also use constexpr int32_t's.